packages/cfd/src/Components/cfd-account-card.tsx

```diff
@@ -259,6 +263,38 @@ const CFDAccountCardComponent = ({
         return '';
     }, []);
 
+    const getBannerStatus = (account: DetailsOfEachMT5Loginid) => {
+        const { landing_company_short, status } = account;
+        if (landing_company_short && status && ['proof_failed', 'verification_pending'].includes(status)) {
```

Comment:
```suggestion
        if (landing_company_short && ['proof_failed', 'verification_pending'].includes(status)) {
```

Reply:
status comes from API. it chan be null| undefined or string. So we need to check is status is valid